rdsbroker/broker.go

```diff
@@ -166,6 +175,14 @@ func (b *RDSBroker) Update(instanceID string, details brokerapi.UpdateDetails, a
 	}
 
 	modifyDBInstance := b.modifyDBInstance(instanceID, servicePlan, updateParameters, details)
+
+	if servicePlan.RDSProperties.ConfigureAllocatedStorage && updateParameters.AllocatedStorage > 0 {
```

Comment:
Could this code be moved to `modifyDBInstance` method?
It could reuse the code from the `createDBInstance` method as commented above.